hardhat.config.ts

```diff
@@ -35,6 +35,12 @@ const config: HardhatUserConfig = {
       accounts:
         process.env.PRIVATE_KEY !== undefined ? [process.env.PRIVATE_KEY] : [],
     },
+    hardhat: {
+      mining: {
```

Comment:
Turning on more traditional mining for the hardhat node allows the Validator to listen for to event logs.

Reply:
nice find